cli/bpmetadata/cmd.go

```diff
@@ -170,8 +207,21 @@ func createInfo(bpPath string, readmeContent []byte) (*BlueprintInfo, error) {
 	return i, nil
 }
 
-func createContent(bpPath string, rootPath string, readmeContent []byte) BlueprintContent {
-	var content BlueprintContent
+func createInterfaces(bpPath string, interfaces *BlueprintInterface) (*BlueprintInterface, error) {
+	i, err := getBlueprintInterfaces(bpPath)
+	if err != nil {
+		return nil, err
+	}
+
+	if interfaces.VariableGroups != nil {
+		i.VariableGroups = interfaces.VariableGroups
+	}
+
```

Comment:
Whats the reasoning for checking incoming interfaces?

Reply:
If the user had added variables groups manually; we should preserve those entries.